packages/compartment-mapper/src/types.js

```diff
@@ -165,8 +165,7 @@ export {};
  */
 
 /**
- * @typedef {ReadPowers | object} MaybeReadPowers
- * @property {MaybeReadFn} maybeRead
+ * @typedef {ReadPowers & {maybeRead: MaybeReadFn}} MaybeReadPowers
```

Comment:
The old pattern broken in general? Used to be vogue.

Reply:
I don't recall that ever working.  To combine types in a `@typedef`, I've always needed to use intersections or something akin to `type-fest`'s `Merge`.

As written, it looks like it _might_ work, but for the wrong reasons--afaict `ReadPowers | object` simplifies to `object`.